core/templates/pages/topic-editor-page/navbar/topic-editor-navbar-breadcrumb.directive.ts

```diff
@@ -29,12 +30,18 @@ angular.module('oppia').directive('topicEditorNavbarBreadcrumb', [
         '/pages/topic-editor-page/navbar/' +
         'topic-editor-navbar-breadcrumb.directive.html'),
       controller: [
-        '$scope', 'TopicEditorStateService',
-        'FocusManagerService', 'TOPIC_NAME_INPUT_FOCUS_LABEL',
+        '$scope', 'TopicEditorStateService', 'TopicEditorRoutingService',
         function(
-            $scope, TopicEditorStateService,
-            FocusManagerService, TOPIC_NAME_INPUT_FOCUS_LABEL) {
+            $scope, TopicEditorStateService, TopicEditorRoutingService) {
           var ctrl = this;
+          $scope.enableBackToTopic = function() {
```

Comment:
This doesn't really "enable" anything right? Can we rename to 'canNavigatetoTopicPage' or similar?

Reply:
Done.